src/main.cpp

```diff
@@ -39,6 +39,7 @@ int main(int argc, char *argv[])
 #endif
     bool verifyexit = false;
     bool check_all_mods = false;
+    bool explicit_seed = false;
```

Comment:
The old flow where there was a seed string that was set and then passed to the seeding function was preferable.